src/pages/signIn/index.jsx

```diff
@@ -1,11 +1,18 @@
 import React from "react";
+<<<<<<< HEAD
+import signInImage from "../../assets/img/Rectangle 254.png";
+import nigeriaFlag from "../../assets/img/Group.png";
+import { emailValidator, passwordValidator } from "../../utils/LoginAuth";
+=======
 // import SignInImage from "../../assets/img/Sign-in-img.png";
 // import Flag from "../../assets/img/Flag.png";
 import {
   emailValidator,
   passwordValidator,
 } from "../../components/LoginRegexValidator";
```

Comment:
- [ ] I get this error on the console which is coming from this component. Kindly check it out 

![image](https://user-images.githubusercontent.com/63473584/203521275-963fd2db-1a06-4a5c-bbd7-5ac4ba2d33c9.png)

Reply:
> * [ ]  I get this error on the console which is coming from this component. Kindly check it out
> 
> 
> ![image](https://user-images.githubusercontent.com/63473584/203521275-963fd2db-1a06-4a5c-bbd7-5ac4ba2d33c9.png)

This is not coming from my page. It's from the profile page, please check well.